schedule.py

```diff
@@ -8,26 +8,47 @@ class Schedule:
     dao = ScheduleDAO()
 
     def __init__(self, document: dict):
+        self.id = str(document['_id']) if '_id' in document else None
         self.owner = document['owner'] if 'owner' in document else None
         self.events = document['events'] if 'events' in document else []
 
     @classmethod
-    def get(cls, owner):
-        sched_dict = cls.dao.get(owner)
+    def get(cls, owner=None):
+        sched_dicts = cls.dao.get(owner)
+        if sched_dicts is None:
```

Comment:
```py
if not isinstance(dict, sched_dicts):
  return None
```

otherwise if anything other than None, is sent back you will get an iteration error on line 20